.travis.yml

```diff
@@ -1,17 +1,15 @@
+
+# This config file is generated by ./scripts/gen_travis.py.
+# Do not edit by hand.
+
 language: generic
 dist: focal
 
-matrix:
+jobs:
   include:
     - os: linux
       arch: amd64
       env: CC=gcc CXX=g++ COMPILER_FLAGS="" CONFIGURE_FLAGS="" EXTRA_CFLAGS="-Werror -Wno-array-bounds"
-    - os: osx
```

Comment:
Question: is it true that only the order of the jobs changed, not the actual combination?

Reply:
Never mind; just saw the PR summary.